add merge and publication policy (#2363)
auto-wcag/auto-wcag: pages/design/process.md

  - went through Call for Review (if applicable), and change requests have been resolved
  - been merged and published as a proposed rule in the WAI website at https://www.w3.org/WAI/standards-guidelines/act/rules/ (ACT Rules on the WAI website are generally updated every Tuesday)

### Merge and publication policy

> To give implementors of ACT Rules time to update implementation data, pull requests cannot be merged on Friday, Saturday, or Sunday. Those three days are reserved for implementors to update their implementation data (automated or otherwise). The publication cycle works as follows:

Reviewer: What happens if something gets merged on Monday, or on Tuesday, right before the WAI website update goes live? As I said yesterday, I think we should restrict the updates coming from the act-rules.github.io repo to a specific date and time during the week, which could be Thursdays after the meeting when hopefully all has been merged.

1. Pull requests in [act-rules/act-rules.github.io][] can be merged Monday through Thursday.
2. Implementors have Friday, Saturday or Sunday to update implementation data.

> 3. Monday morning a publication candidate is created as a pull request in [w3c/wcag-act-rules][], for review by WAI staff.

Reviewer: ```suggestion
3. Monday morning, a pull request for updating the WAI website is created in [w3c/wcag-act-rules][], for review by WAI staff.
```

> 4. Somewhere that week, usually Monday or Tuesday, the publication candidate is approved and published to the WAI website.

Reviewer: ```suggestion
4. Sometime during the week, usually Monday or Tuesday, the pull request is approved and published to the WAI website.
```

Reviewer 2: ```suggestion
4. The publication candidate is approved and published to the WAI website. This usually happens sometime during the week, typically Monday or Tuesday. For more information, see the [publication workflow](https://wai-website-theme.netlify.app/workflow/).
```

We strive to (and usually) approve and publish during the week, but sometime it is delayed.
I also think good to point to the [publication process](https://wai-website-theme.netlify.app/workflow/).

> For substantial changes, implementors may request a publication be skipped to allow extra time for implementation. This should be done on the publication candidate pull request. It is up to discretion of W3C staff to decide whether the publication is skipped.

Reviewer: ```suggestion
For substantial changes, implementors may request that a publication be skipped to allow extra time for implementation. This should be done on the WAI website pull request. It is at the discretion of W3C staff to decide whether the publication is skipped.
```

Reviewer 2: ```suggestion
For substantial changes, implementors may request that a publication be skipped to allow extra time for implementation. This should be done on the pull request in [w3c/wcag-act-rules][]. It is up at the discretion of W3C staff to decide whether the publication is skipped.
```

Minor. @daniel-montalvo please note that I've included your suggested changes in my suggestion.

Reviewer: Thanks @remibetin I think it should be "at the discretion" instead of "up at the discretion". Otherwise I like your suggestion better :-)
